CMakeLists.txt

```diff
@@ -103,8 +104,26 @@ if(OCIO_BUILD_GPU_TESTS OR OCIO_BUILD_APPS)
         message(WARNING "GLUT not found.")
         set(OCIO_GL_ENABLED OFF)
     endif()
+
+    if(${OCIO_USE_HEADLESS})
+        set(OCIO_EGL_HEADLESS ON)
+        # Find ECM package to use FindEGL.cmake
+        find_package(ECM REQUIRED NO_MODULE)
```

Comment:
Looking into the release note of `CMake`, I believe neither `CMake` 3.12 nor 3.10 has `findEGL.cmake` by default. One solution is to ship `findEGL.cmake` with OCIO (put the file in /share/cmake/modules/). Is this acceptible?

Reply:
Hello, standard `FindOpenGL.cmake` module supports EGL since 3.10. It sets appropriate variables and creates imported targets (`OpenGL::GL`, `OpenGL::EGL`).

Reply:
Reference: https://cmake.org/cmake/help/v3.12/module/FindOpenGL.html